src/include/daos_srv/rebuild.h

```diff
@@ -35,6 +35,37 @@ typedef enum {
 			  (rb_op) == RB_OP_NONE ? "None" : \
 			  "Unknown")
 
+/* Common rebuild identifying information for INFO/DEBUG logging:
+ * rb=<pool_uuid>/<rebuild_ver>/<term>/<rebuild_gen>/<ldr_engine_rank>/<opstring>
+ */
+#define DF_RB  "rb=" DF_UUID "/%u/" DF_U64 "/%u/%u/%s"
+
+/* Full rebuild identifying information includes engine_rank:tgt_id */
+#define DF_RBF DF_RB " r:t=%u:%d"
+
+/* arguments for log rebuild identifier given a struct rebuild_global_pool_tracker * */
+#define DP_RB_RGT(rgt)                                                                             \
+	DP_UUID((rgt)->rgt_pool_uuid), (rgt)->rgt_rebuild_ver, (rgt)->rgt_leader_term,             \
+	    (rgt)->rgt_rebuild_gen, dss_self_rank(), RB_OP_STR((rgt)->rgt_opc)
```

Comment:
thanks, D_LOG_FILE_APPEND_RANK can be used to append #rank to the log file name, is it convenient for you to use? Some times a failure happened, but when we enable the log mask it cannot be reproduce anymore, so we may need to consider to log as less as possible if it can prove enough information.